### PR TITLE
httpx: new, 0.13.3, along with its dependencies

### PR DESCRIPTION
**Pull request by OriginCode**
https://pakreq.aosc.io/detail/698

```
httpx: 0.13.3
hstspreload: 20200717
httpcore: 0.9.1
rfc3986: 1.4.0
sniffio: 1.1.0
```

### CODE REVIEW

**Review by liushuyu — COMMENTED** [review]
The patch file `0001-ca-certs.patch` needs a more descriptive name, like "use-system-ca-certs" or something like that.

**Comment by OriginCode** [comment]
> The patch file `0001-ca-certs.patch` needs a more descriptive name, like "use-system-ca-certs" or something like that.

https://github.com/AOSC-Dev/aosc-os-abbs/pull/2213/commits/99019bbda4b70d766e1893922832434a56c00033

**Review by liushuyu — APPROVED** [review]
LGTM

File: extra-python/hstspreload/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,8 @@
+PKGNAME=hstspreload
+PKGSEC=python
+PKDDEP="python-3"
+BUILDDEP="setuptools"
+PKGDES="Chromium HSTS Preload list"
```
Comment (MingcongBai) [inline extra-python/hstspreload/autobuild/defines]: This description is not exactly descriptive. Although summarised as such on its original [PyPI page](https://pypi.org/project/hstspreload/) (which I suspect is where you got this from). I would suggest changing this.

File: extra-python/hstspreload/spec [hunk 2]
```diff
@@ -0,0 +1,3 @@
+VER=20200717
+SRCTBL="https://github.com/sethmlarson/hstspreload/archive/2020.7.17.tar.gz"
```
Comment (MingcongBai) [inline extra-python/hstspreload/spec]: Hard-coded version in source link, please use `2020.7.17` as version number (why did you bother changing this?).

File: extra-python/httpcore/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,8 @@
+PKGNAME=httpcore
+PKGSEC=python
+PKGDEP="hyper-h11 hyper-h2 sniffio"
+BUILDDEP="setuptools"
+PKGDES="A minimal HTTP client."
```
Comment (MingcongBai) [inline extra-python/httpcore/autobuild/defines]: Trim period point here.

File: extra-python/httpx/autobuild/defines [hunk 4]
```diff
@@ -0,0 +1,8 @@
+PKGNAME=httpx
+PKGSEC=python
+PKGDEP="chardet hstspreload sniffio idna rfc3986 httpcore"
+BUILDDEP="setuptools pathlib"
+PKGDES="A next generation HTTP client for Python"
```
Comment (MingcongBai) [inline extra-python/httpx/autobuild/defines]: For Python? Or written in Python?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-python/hstspreload/autobuild/defines (hunk 1 above)
Comment (OriginCode) [reply extra-python/hstspreload/autobuild/defines]: 789c19a

File: extra-python/hstspreload/spec (hunk 2 above)
Comment (OriginCode) [reply extra-python/hstspreload/spec]: 789c19a

File: extra-python/httpcore/autobuild/defines (hunk 3 above)
Comment (OriginCode) [reply extra-python/httpcore/autobuild/defines]: 908bdd1

File: extra-python/httpx/autobuild/defines (hunk 4 above)
Comment (OriginCode) [reply extra-python/httpx/autobuild/defines]: 954add2ceb

**Review by MingcongBai — APPROVED** [review]
Great!

**Pull request merged**